examples/case-study-1/load-db.p6

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env perl6
+
+use Red;
+
+use lib <./lib>;
```

Comment:
In order to make it work as ```perl6 -I. examples/case-study-1/load-db.p6 go``` I would change this to:

```perl6
use lib $*PROGRAM.parent.add('lib');
```

Reply:
so done